doc/rst/technotes/localeModels.rst

```diff
@@ -104,14 +104,14 @@ when using the flat locale model, and often worse.  The *multi-ddata*
 feature introduced in the 1.15 Chapel release improved some cases for
 the NUMA locale model but slowed many others, sometimes by a lot.  In
 the end we disabled it because much of the performance loss was inherent
-in the implementation and could not be removed.  On Cray XE and XC
-systems with ``CHPL_COMM=ugni`` and NIC-registered memory, recent work
-to allocate arrays separately and register them dynamically has improved
-NUMA affinity and thus performance, but the benefits of that effort
-apply to the flat locale model as well as they do to the numa one.  For
-other configurations, with the multi-ddata feature disabled performance
-with the NUMA locale model has returned, for better or worse, to what it
-was before that was introduced.  At present most of our effort has to do
+in the implementation and could not be removed.  On Cray XC systems with
```

Comment:
The multi-ddata feature is not just disabled as we say below, but removed and won't return.  I think we could simply remove the two sentences "The multi-ddata feature ... and could not be removed." here, and also the last two in this paragraph "For other configurations, ... first-touch to achieve NUMA affinity."